.ci/setup_env_ubuntu.sh

```diff
@@ -43,6 +43,8 @@ declare -A packages=( \
 	[redis]="redis-server" \
 )
 
+rust_agent_pkgs="build-essential g++ make cmake automake autoconf m4 libc6-dev libstdc++-8-dev coreutils binutils debianutils gcc musl musl-dev musl-tools git"
```

Comment:
How about:

```sh
rust_agent_pkgs=()
rust_agent_pkgs+=(build-essential)
rust_agent_pkgs+=(g++)
rust_agent_pkgs+=(make)
```

That makes it easier to read the list, easier to diff and easier to sort alphabetically :smile:

Reply:
fixed